pkg/resource/certificate_authority/custom_delta.go

```diff
@@ -16,25 +16,18 @@ package certificate_authority
 import (
 	svcapitypes "github.com/aws-controllers-k8s/acmpca-controller/apis/v1alpha1"
 	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
-	"github.com/aws/aws-sdk-go/aws"
 )
 
 func customSetDefaults(
 	a *resource,
 	b *resource,
```

Comment:
can you update these names to be more descriptive?

Reply:
we are just going to leave this the same as this is the standard naming scheme across all ACK controllers.